<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditObjectDialog</class>
 <widget class="QDialog" name="EditObjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Терминал:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLineEdit" name="lineEditPhoneAZS">
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Телефон директора</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>E-mail</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLineEdit" name="lineEditAdress_2">
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLineEdit" name="lineEditPhoneDir">
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Расположение:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Телефон АЗС</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditTerminalID">
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="lineEditAdress">
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>150</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="labelClientName">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditTerminalID</tabstop>
  <tabstop>lineEditAdress</tabstop>
  <tabstop>lineEditPhoneAZS</tabstop>
  <tabstop>lineEditPhoneDir</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
